pkg/handler/handler.go

```diff
@@ -40,13 +41,36 @@ import (
 )
 
 const (
-	healthCheckAddr = ":8081"
-	livenessPath    = "/healthz"
-	readinessPath   = "/readyz"
-	metricsPath     = "/metrics"
-	whHandlePath    = "/validate/v1alpha1/blueprint"
+	livenessPathEnvVar  = "LIVENESS_PATH"
+	readinessPathEnvVar = "READINESS_PATH"
+	healthPortEnvVar    = "HEALTH_PORT"
+	metricsPath         = "/metrics"
+	whHandlePath        = "/validate/v1alpha1/blueprint"
 )
 
+var (
+	healthCheckAddr string
+	livenessPath    string
+	readinessPath   string
+)
+
+func init() {
+	// set default values for health checks.
+	healthCheckAddr = ":8000"
```

Comment:
ni. Should we only define the port and separate const for default healthCheckAddr?
Also, was change of default port from 8081 to 8000 accidental or intentional